pkg/loadtester/loadtester.go

```diff
@@ -140,29 +141,47 @@ func (t *LoadTester) PublishTrack(name string, kind lksdk.TrackKind, bitrate uin
 		return p.SID(), nil
 	}
 
-	// video
-	sampleProvider, err := NewLoadTestProvider(bitrate)
-	if err != nil {
-		return "", err
-	}
-	track, err := lksdk.NewLocalSampleTrack(webrtc.RTPCodecCapability{
-		MimeType:    webrtc.MimeTypeVP8,
-		ClockRate:   33,
-		SDPFmtpLine: "",
-		RTCPFeedback: []webrtc.RTCPFeedback{
-			{Type: webrtc.TypeRTCPFBNACK},
-			{Type: webrtc.TypeRTCPFBNACK, Parameter: "pli"},
-		},
-	})
-	if err != nil {
-		return "", err
-	}
-	if err := track.StartWrite(sampleProvider, nil); err != nil {
-		return "", err
+	var tracks []*lksdk.LocalSampleTrack
+
+	// for video, publish three simulcast layers
+	for i := livekit.VideoQuality_LOW; i <= livekit.VideoQuality_HIGH; i++ {
+		// scale by 1, 2, 4
+		scaleBy := uint32(math.Pow(2, 2-float64(i)))
+		layer := &livekit.VideoLayer{
+			Quality: i,
+			// bitrate scales by /4, /16
+			Bitrate: bitrate / (scaleBy * scaleBy),
+			Width:   highWidth / scaleBy,
+			Height:  highHeight / scaleBy,
+		}
+		fmt.Println("using simulcast quality ", i, "bitrate", layer.Bitrate)
+
+		sampleProvider, err := NewLoadTestProvider(layer.Bitrate)
+		if err != nil {
+			return "", err
+		}
+
+		track, err := lksdk.NewLocalSampleTrack(webrtc.RTPCodecCapability{
+			MimeType:    webrtc.MimeTypeVP8,
+			ClockRate:   33,
```

Comment:
Why is the clock rate 33 here and not 90K?

Reply:
Looks like we have to use frame interval in ms. Guess that is how that thing works. I see audio also set to 20.